x-pack/plugins/security/public/components/management/users/users.js

```diff
@@ -69,7 +75,13 @@ export class Users extends Component {
         color="danger"
         onClick={() => this.setState({ showDeleteConfirmation: true })}
       >
-        Delete {numSelected} user{numSelected > 1 ? 's' : ''}
+        <FormattedMessage
+          id="xpack.security.components.management.users.deleteUsersButtonLabel"
+          defaultMessage="Delete {numSelected} user{numSelected, plural, one { } other {s}}"
```

Comment:
Could you please add a screenshot how a label looks like on UI when `numSelected` equals 1?

Reply:
![image](https://user-images.githubusercontent.com/23155410/47639136-9a38c800-db71-11e8-8856-4bfd5808554d.png)